pydy/viz/server.py

```diff
@@ -1,37 +1,107 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
We shouldn't do this. Just make sure to use parentheses on all print statements.